src/Common/Version.props

```
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <AssemblyVersion>1.13.0</AssemblyVersion>
+    <AssemblyVersion>1.14.0</AssemblyVersion>
```

This is managed by the team during the release.

```suggestion
    <AssemblyVersion>1.13.0</AssemblyVersion>
```

---

fixed